### ISSUE TITLE
PdhLookupPerfIndexByEnglishName causes ExceptionInInitializerError

### ISSUE THREAD

**Issue by TheXaver** [oshi/oshi#762]
Exception in thread "GetHWID Thread" java.lang.ExceptionInInitializerError
        at oshi.hardware.platform.windows.WindowsHardwareAbstractionLayer.getDiskStores(Unknown Source)
        at ru.gravit.launcher.s6cWIgQFaKFozpiF0rvCUkV4.tIxukqwJpjbqRiR8fRRJ9oyw(
Unknown Source)
        at ru.gravit.launcher.s6cWIgQFaKFozpiF0rvCUkV4.OvndpPHp1rqtyiydNxBfVvo1(
Unknown Source)
        at ru.gravit.launcher.DJBo2tP0OoFaOZANQ00ZcxtJ.getHWID(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
Caused by: com.sun.jna.platform.win32.Win32Exception: Не удается найти указанный
 файл.
        at com.sun.jna.platform.win32.Advapi32Util.registryGetStringArray(Unknow
n Source)
        at com.sun.jna.platform.win32.Advapi32Util.registryGetStringArray(Unknow
n Source)
        at com.sun.jna.platform.win32.Advapi32Util.registryGetStringArray(Unknow
n Source)
        at com.sun.jna.platform.win32.PdhUtil.PdhLookupPerfIndexByEnglishName(Un
known Source)
        at oshi.hardware.platform.windows.WindowsDisks.<clinit>(Unknown Source)

**Comment by TheXaver** [comment]
Windows 8.1

**Comment by dbwiddis** [comment]
The English Name lookup relies on the registry containing counters in the [standard PerfCounter location](https://docs.microsoft.com/en-us/windows/desktop/PerfCtrs/adding-counter-names-and-descriptions-to-the-registry) in the registry, `HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Perflib\009`.  Your system may not have these English counters in the correct location.

You may need to [manually rebuild the PerfMon Counter library ](https://support.microsoft.com/en-us/help/300956/how-to-manually-rebuild-performance-counter-library-values) values.

Please check your registry and see if you have the `\009` key.
```
HKEY_LOCAL_MACHINE
   \SOFTWARE
      \Microsoft
         \Windows NT
            \CurrentVersion
               \Perflib
                  Last Counter = highest counter index
                  Last Help = highest help index
                  \009
                     Counters = 2 System 4 Memory...
                     Help = 3 The System Object Type...
                  \supported language, other than English
                     Counters = ...
                     Help = ...
```

**Comment by dbwiddis** [comment]
(That said, OSHI should have a more sane fallback to WMI when this error occurs and I'll work that as part of #755, the one class that missed my earlier attempt to get rid of these static initializer issues.)

**Comment by TheXaver** [comment]
I will check.

**Comment by dbwiddis** [comment]
Meanwhile I've made a quick change in the 3.12.X branch (#764) which is available on maven as 3.12.3-SNAPSHOT, that will avoid the exception and fall back to the WMI counters.

**Comment by Sirse** [comment]
Can't get 3.12.3-SNAPSHOT from mvn. Release-only repository?

**Comment by dbwiddis** [comment]
You can download the jar [here](https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=com.github.oshi&a=oshi-core&v=3.12.3-SNAPSHOT&e=jar).

You may have to [configure maven to download snapshots](https://stackoverflow.com/questions/7715321/how-to-download-snapshot-version-from-maven-snapshot-repository).

**Comment by dbwiddis** [comment]
Also that whole class has had major rewrites on the 4.0.0 snapshot / master branch, if you're on Java 8.

**Comment by dbwiddis** [comment]
This fix, and the major rewrites in 4.0.0, have been included in the 3.13.0 release.
